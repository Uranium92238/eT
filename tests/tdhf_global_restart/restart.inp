system
   name: HOF He
   charge: 0
end system

do
   restart
   tdhf excited state
end do

memory
   available: 8
end memory

solver scf
   algorithm:          scf-diis
   energy threshold:   1.0d-10
   gradient threshold: 1.0d-10
end solver scf

method
   hf
end method

solver tdhf es
  singlet states: 2
  residual threshold: 1.0d-10
end solver tdhf es

geometry
basis: cc-pVDZ
 H          0.86681        0.60144        5.00000
 H         -0.86681        0.60144        5.00000
 O          0.00000       -0.07579        5.00000
 He         0.00000        0.00000        7.50000
end geometry
