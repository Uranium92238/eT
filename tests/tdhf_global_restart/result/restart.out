


                     eT 1.9 - an electronic structure program

  ------------------------------------------------------------------------
   Author list in alphabetical order:
  ------------------------------------------------------------------------
   J. H. Andersen, A. Balbi, S. Coriani, S. D. Folkestad, T. Giovannini, 
   L. Goletto, T. S. Haugland, A. Hutcheson, I-M. Høyvik, E. F. Kjønstad, 
   H. Koch, R. Matveeva, T. Moitra, R. H. Myhre, A. C. Paul, S. Roet, 
   E. Ronca, M. Scavino, A. K. Schnack-Petersen, A. S. Skeidsvoll, Å. 
   H. Tveten
  ------------------------------------------------------------------------
   J. Chem. Phys. 152, 184103 (2020); https://doi.org/10.1063/5.0004713


   This is eT 1.9.0 Jazzy
  ------------------------------------------------------------
  Configuration date: 2022-10-19 10:25:53 UTC -07:00
  Git branch:         detached HEAD
  Git hash:           57c35dcc31df636729251e2448b7b0d0dd22e1f1
  Fortran compiler:   GNU 11.3.0
  C compiler:         GNU 11.3.0
  C++ compiler:       GNU 11.3.0
  LAPACK type:        SYSTEM_NATIVE
  BLAS type:          SYSTEM_NATIVE
  64-bit integers:    OFF
  OpenMP:             ON
  PCM:                OFF
  Forced batching:    OFF
  Runtime checks:     OFF
  ------------------------------------------------------------


  :: Input file
  ================

     Note: geometry section is excluded from this print

     system
        name: hof he
        charge: 0
     end system

     do
        restart
        tdhf excited state
     end do

     memory
        available: 8
     end memory

     solver scf
        algorithm:          scf-diis
        energy threshold:   1.0d-10
        gradient threshold: 1.0d-10
     end solver scf

     method
        hf
     end method

     solver tdhf es
       singlet states: 2
       residual threshold: 1.0d-10
     end solver tdhf es


  Calculation start:2022-10-19 10:30:03 UTC -07:00

  Running on 2 OMP threads
  Memory available for calculation: 8.000000 GB


  :: RHF wavefunction
  ===================

     ==============================================================================
                                     Geometry (angstrom)
     ==============================================================================
        Atom           X                  Y                  Z         # in input
     ==============================================================================
        Basis: cc-pvdz
        1  H     0.866810000000     0.601440000000     5.000000000000        1
        2  H    -0.866810000000     0.601440000000     5.000000000000        2
        3  O     0.000000000000    -0.075790000000     5.000000000000        3
        4 He     0.000000000000     0.000000000000     7.500000000000        4
     ==============================================================================

     ==============================================================================
                                     Geometry (a.u.)
     ==============================================================================
        Atom           X                  Y                  Z         # in input
     ==============================================================================
        Basis: cc-pvdz
        1  H     1.638033502034     1.136556880358     9.448630622825        1
        2  H    -1.638033502034     1.136556880358     9.448630622825        2
        3  O     0.000000000000    -0.143222342981     9.448630622825        3
        4 He     0.000000000000     0.000000000000    14.172945934238        4
     ==============================================================================

  - Cholesky decomposition of AO overlap to get linearly independent AOs:

     Linear dependence threshold:             0.10E-05
     Number of atomic orbitals:               29
     Number of orthonormal atomic orbitals:   29

  - Molecular orbital details:

     Number of occupied orbitals:         6
     Number of virtual orbitals:         23
     Number of molecular orbitals:       29


  Determining reference state
  ===========================

  - Requested restart. Reading orbitals from file

  - Screening and integral thresholds:

     Coulomb screening threshold:    0.1000E-15
     Exchange screening threshold:   0.1000E-13
     ERI cutoff:                     0.1000E-15
     One-electron integral  cutoff:  0.1000E-20
     Cumulative Fock threshold:      0.1000E+01

  - SCF solver settings:

     Maximum iterations:                   100
     Acceleration type:                   diis

  - Convergence thresholds

     Residual threshold:            0.1000E-09
     Energy threshold:              0.1000E-09

  - DIIS tool settings:

     DIIS dimension:   8

     Storage (solver scf_errors): memory
     Storage (solver scf_parameters): memory

  Iteration       Energy (a.u.)      Max(grad.)    Delta E (a.u.)
  ---------------------------------------------------------------
     1           -78.843646205256     0.2324E-10     0.7884E+02
  ---------------------------------------------------------------
  Convergence criterion met in 1 iterations!

  - Summary of RHF wavefunction energetics (a.u.):

     HOMO-LUMO gap:                  0.645693538823
     Nuclear repulsion energy:      12.167611407170
     Electronic energy:            -91.011257612426
     Total energy:                 -78.843646205256


  Determining TDHF excitation energies
  ====================================
     Storage (tdhf_davidson_trials): memory
     Storage (tdhf_davidson_transforms): memory

  Davidson eigenvalue equation solver
  -----------------------------------

  A Davidson solver that solves an eigenvalue equation: M x = omega x. 
  This equation is solved in a reduced space. A description of the algorithm 
  can be found in E. R. Davidson, J. Comput. Phys. 17, 87 (1975).

   - Davidson tool settings:

     Number of parameters:                  138
     Number of requested solutions:           2
     Max reduced space dimension:            50


  Requested restart. Reading in solution from file.

  Requested restart. Reading in solution from file.

  - Davidson solver settings

     Number of singlet states:               2
     Max number of iterations:             100

  Iteration:                  1
  Reduced space dimension:    2

  Root  Eigenvalue (Re)   Eigenvalue (Im)    |residual|   Delta E (Re)
  -------------------------------------------------------------------------
     1   0.078189601095    0.000000000000     0.1387E-10   0.7819E-01
     2   0.142327765331    0.000000000000     0.2414E-10   0.1423E+00
  -------------------------------------------------------------------------
  Convergence criterion met in 1 iterations!

  - TDHF excitation energies:

                                      Excitation energy
                          ------------------------------------------
      State                (Hartree)             (eV)
     ---------------------------------------------------------------
        1                  0.279624035259        7.608957563911
        2                  0.377263522396       10.265863339180
     ---------------------------------------------------------------
     eV/Hartree (CODATA 2014): 27.21138602

  Peak memory usage during the execution of eT: 213.148 KB

  Total wall time in eT (sec):              0.10300
  Total cpu time in eT (sec):               0.16420

  Calculation end:2022-10-19 10:30:03 UTC -07:00

  - Implementation references:

     eT: https://doi.org/10.1063/5.0004713

  eT terminated successfully!
